### ISSUE TITLE
Add Brunei Darussalam holidays

### ISSUE THREAD

**Issue by arkid15r** [dr-prodigy/python-holidays#1157]
Useful links:
  - https://en.wikipedia.org/wiki/Public_holidays_in_Brunei
  - https://www.timeanddate.com/holidays/brunei/

**Comment by PPsyrius** [comment]
I'm working on this at the moment, though I've yet to find the relevant legal clause on in-lieu holidays being observed on the following Monday yet based on ASEAN's [2022](https://asean.org/wp-content/uploads/2021/12/ASEAN-National-Holidays-2022.pdf) and [2023](https://asean.org/wp-content/uploads/2022/12/ASEAN-Public-Holidays-2023.pdf) calendar.

**Comment by PPsyrius** [comment]
Gonna move the to-do codes here for possible #1168 extensions should anyone be up for it:
```python
# Former Holidays
# This section is here should we extend our calendar coverage later on.

# Good Friday
# Good Friday
# Status: No longer observed post-Independence.

# Freedom Day
# Hari Kebebasan
# Status: No longer observed.
# Commemorating the landing of the Allies in Muara to liberate Brunei
#   on June 10, 1945. Presumed to be observed from 1946-1983.

# Queen Elizabeth II's Birthday
# Hari Lahir Queen
# Status: No longer observed.
# Observed on June, 12 from 1952-1983.

# Sultan Omar Ali Saifuddien III's Birthday
# Hari Lahir Duli Yang Maha Mulia
# Status: No longer observed.
# Observed on September, 23 from 1950-1967.

# Safar Bath
# Mandi Safar
# Status: No longer observed.
# Observed on last Wednesday on the month of Safar until around 1963.
```

**Comment by KJhellico** [comment]
Implemented in #1168.
